modules/visual_script/doc_classes/VisualScriptCustomNode.xml

```diff
@@ -123,7 +159,7 @@
 			</argument>
 			<description>
 				Execute the custom node's logic, returning the index of the output sequence port to use or a [String] when there is an error.
-				The [code]inputs[/code] array contains the values of the input ports.
+				The [code]inputs[/code] array contains the values of the input port's.
```

Comment:
Ouch, I missed that one. This one is adding a new typo.

"ports" is the plural of "port".
"port's" is the possessive form, equivalent to "of the port".

Reply:
This one is still wrong, the others are fixed.

Reply:
Ohh, ...
I thot it to be an english vs american thin.

Getting comfortable with force push is dangerous.
Almost messed up another PR but cathed it in time.